src/pseudo/load_psp.jl

```diff
@@ -42,3 +40,7 @@ function load_psp(key::AbstractString)
               "'$identifier' in directory '$(datadir_psp())'")
     end
 end
+
+function load_psp(dir::AbstractString, filename::AbstractString)
+    load_psp(joinpath(dir, filename))
+end
```

Comment:
Why is this needed? Could one not just use the `@artifact` macro like so:
```julia
load_psp(@artifact"pd_nc_sr_lda_standard_0.4.1_upf/Si.upf")
```

Reply:
Ah, I didn't know there was a macro or that it behaved differently from the string magic.
Did this work for you locally?

Reply:
I've tried this in the newest commit now.

Reply:
Turns out you need to remove the `@`, so `load_psp(artifact"pd_nc_sr_lda_standard_0.4.1_upf/Si.upf")` works.